assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/workers/workers.tf

```diff
@@ -67,6 +77,11 @@ data "ct_config" "ignitions" {
       setup_raid_hdd        = var.setup_raid_hdd
       setup_raid_ssd        = var.setup_raid_ssd
       setup_raid_ssd_fs     = var.setup_raid_ssd_fs
+
+      // This variable is not really used in the template, it is just to create
+      // a dependency from packet_device.nodes (that uses this a user_data input
+      // param) to the resources listed on var.depends_on_hack
+      //depends_on_hack = join(", ", var.depends_on_hack)
```

Comment:
Everything commented out?

Reply:
Left over :see_no_evil:

Reply:
Fixed locally, will be included in the next push :)